tests/acceptance/live-test.js

```diff
@@ -24,24 +24,35 @@ module('Acceptance | live', function (hooks) {
     // await click('[data-test-button="live-join"]');
     // TODO: Fix and improve this
     /*
-      Line 30 has been commented as currently the click serves no meaning, as the test is incomplete and
-      the button click fires a network call which causes test to fail
-
-      Currently the when the button is clicked it makes a api call, which fails after some time
-      When a promise was fired by this test, but this test don't wait for it to get resolved before this test is torn down,
-      So later when the promise is resolved/rejected there no reference to the tests which had fired the event, so it gives error
+    Line 30 has been commented as currently the click serves no meaning, as the test is incomplete and
+    the button click fires a network call which causes test to fail
+    
+    Currently the when the button is clicked it makes a api call, which fails after some time
+    When a promise was fired by this test, but this test don't wait for it to get resolved before this test is torn down,
+    So later when the promise is resolved/rejected there no reference to the tests which had fired the event, so it gives error
+    
+    **ERROR message
+    global failure: Error: Cannot call `.lookup` after the owner has been destroyed@ 30 ms
+    Source:
+    
+    lookup@http://localhost:4200/assets/vendor.js:11608:15
+    lookup@http://localhost:4200/assets/vendor.js:33127:33
+    getInjection@http://localhost:4200/assets/vendor.js:23778:20
+    get/<@http://localhost:4200/assets/vendor.js:21679:25
+    untrack@http://localhost:4200/assets/vendor.js:61174:14
+    
+    This can be solved by waiting for the promise to get resolved before the test is torn down
+    */
+  });
+  test('on button click. when user is changing route', async function (assert) {
+    assert.expect(2);
 
-      **ERROR message
-        global failure: Error: Cannot call `.lookup` after the owner has been destroyed@ 30 ms
-        Source:
+    await visit('/live?dev=true');
+    assert.dom('[data-test-status]').hasText('Status');
 
-        lookup@http://localhost:4200/assets/vendor.js:11608:15
-        lookup@http://localhost:4200/assets/vendor.js:33127:33
-        getInjection@http://localhost:4200/assets/vendor.js:23778:20
-        get/<@http://localhost:4200/assets/vendor.js:21679:25
-        untrack@http://localhost:4200/assets/vendor.js:61174:14
+    await visit('/live');
 
-      This can be solved by waiting for the promise to get resolved before the test is torn down
-    */
+    await click('button');
+    assert.strictEqual(currentURL(), '/live', 'we are not on live page');
   });
```

Comment:
I don't think this test is testing the flow of this PR

Reply:
You are correct. I discussed the implementation of the test for alert in qunit with Ankush. This is not straight-forward and has a learning curve. 

I am asked to close this story and create a new story where developer needs to write a test for alert.